src/framework/uicomponents/qml/MuseScore/UiComponents/StyledTextLabel.qml

```diff
@@ -3,6 +3,8 @@ import QtQuick 2.12
 Text {
     id: root
 
+    readonly property bool isEmpty: Boolean(text)
```

Comment:
You have a bug here. Should be:

readonly property bool isEmpty: !Boolean(text)

or just:
readonly property bool isEmpty: text.length === 0